python-avd/pyavd/_eos_designs/shared_utils/underlay.py

```diff
@@ -91,7 +99,7 @@ def underlay_ipv6_numbered(self: SharedUtilsProtocol) -> bool:
             if self.is_wan_router:
                 msg = "Invalid combination of inputs. WAN is not yet supported with IPv6 underlay"
                 raise AristaAvdInvalidInputsError(msg)
-            if self.underlay_multicast_rp_interfaces or self.underlay_multicast:
+            if self.underlay_multicast_rp_interfaces or self.underlay_multicast_pim_sm_enabled or self.underlay_multicast_static_enabled:
```

Comment:
hmmm wondering if you cannot simplify given that `underlay_multicast_rp_interfaces` requires `underlay_multicast_pim_sm_enabled`